doc/manual/toc.tex

```diff
@@ -1,7 +1,10 @@
 \begingroup
-\fontfamily{ptm}
-\normalsize
-\fontseries{c}\selectfont
+%\fontfamily{phv}
+%\normalsize
+%\fontseries{c}\selectfont
```

Comment:
Why comment out these lines, and why a modified version? This is misleading.

Reply:
Was due to preliminary testing. Commented lines are now removed.